Encountify/ViewModels/LocationDetailViewModel.cs

```diff
@@ -14,6 +15,8 @@ public class LocationDetailViewModel : BaseViewModel
         private double coordX;
```

Comment:
Maybe we should change coordX and coordY here to longitude and latitude as well.

Reply:
Forgot to change it